README.md

```diff
@@ -6,9 +6,10 @@ snappy-java is a Java port of the snappy
 ## Features
   * Fast compression/decompression around 200~400MB/sec.
   * Less memory usage. SnappyOutputStream uses only 32KB+ in default.
-  * JNI-based implementation to achieve comparable performance to the native C++ version.  
+  * JNI-based implementation to achieve comparable performance to the native C++ version.
      * Although snappy-java uses JNI, it can be used safely with multiple class loaders (e.g. Tomcat, etc.).
   * Compression/decompression of Java primitive arrays (`float[]`, `double[]`, `int[]`, `short[]`, `long[]`, etc.)
+     * To improve the compression ratios of these arrays, you can use a fast data-rearrangement implementation ([`BitShuffle`](https://github.com/kiyo-masui/bitshuffle)) before compression
```

Comment:
Link to BitShuffle should be `#bitshuffle-api`. Reference to kiyo-masui/bitshuffle should be elsewhere.